.github/workflows/ci.yml

```diff
@@ -19,58 +19,58 @@
 ---
 name: Airflow
 on:
+  schedule:
+    # Schedule cron jobs every day at 2.10 am
+    - cron: '10 2 * * *'
   push:
-    branches: ['*']
+    branches: ['master', 'v1-10-test', 'v1-10-stable']
```

Comment:
Should we do same for pull request just in case?

Reply:
Yep